accessibility/index.html

```diff
@@ -0,0 +1,556 @@
+					</aside>
+				</section>
+
+				<!-- Exercise 01 END -->
+
+				<!-- Exercise 02 START -->
+
+				<section>
+					<h3>Exercise 2</h3>
+          			<p>Modify the HTML using <strong>Semantic</strong> HTML elements <br/>and read the page again</p>
+		  			<small><a href="https://inclusiveThinking.jsbin.com/dihebu/3/edit?html,css,output" target="_blank">Modify HTML from Exercise One</a></small>
+				</section>
+
+				<!-- Exercise 02 END -->
+
+				<!-- Exercise 03 START -->
+			
+				<section>
+					<h3>Exercise 3</h3>
+          			<p>Use an accessibility testing plugin to test the page.</p>
+                    <small>Instructions for installing WAVE or Tota11y Plugin, from Khan Academy will be given here or at the beginning of the workshop</small>
+				</section>
+
+				<!-- Excercise 03 END -->
+
+				<!-- AUTOMATED TEST RESULTS START -->
+				
+				<section>
+					<h2>We used Sematic HTML elements! <br/>What's with the all the <span style="color: red">errors</span>?</h2>
+
+				<aside class="notes">
+						Semantic elements work as expected when following guideliness and augment with AIRA
+				</aside>
+
+				</section>
+
+				<!-- AUTOMATED TEST RESULTS END -->
+
+				<!-- WCAG 2.0 START -->
+				
+				<section>
+					<h2>Web Content Accessibility Guidelines - <span class="gdi">WCAG 2.0</span></h2>
+					<p>A technical standard made of Four Principles with Twelve Guidelines containing testable Sucess Criteria</p>
+					<p>We will refer to the Web Content Accessibility Guideliness and Success Crtieria in the upcoming excercises</p>
+					<small><a href="http://www.w3.org/TR/UNDERSTANDING-WCAG20/Overview.html" target="_blank"><em>Understanding Web Content Accessibility Guidelines 2.0</em></a><br/>W3C - Web Accessibility Initiative</small>
+				</section>
+
+				<!-- WCAG 2.0 END -->
+            
+                <!-- SUCCESS CRITERIA EXAMPLES START -->
+            
+                <section>
+                    <h2>WCAG <span class="gdi">Success Criteria</span> Examples</h2>
+                    <pre><code data-trim contenteditable> 
+                    
+<!-- SC 1.3.1 -->
+ DOCTYPE html lang="en"
+ 
+ <!-- SC 1.3.1 - SC 2.4.1 - SC 2.4.6 - SC 2.4.10 -->
+ <h1>Page Heading</h1> 
+ 
+ <!-- SC 1.3.1 - SC 2.4.1 - SC 2.4.6 - SC 2.4.10 -->
+ <h2>Join our mailing list</h2>
+
+ <!-- SC 1.3.1 - SC 3.3.3 - SC 4.1.2 -->
+ <label for="email">Please enter your email address</label> 
+ <input type="text" name="email" placeholder="Please enter your email address">
+
+                    </code></pre>
+                    
+                </section>
+            
+                <!-- SUCCESS CRITERIA EXAMPLES END -->
+            
+             <!-- ERROR -->
+            
+            <section>
+                        <small>(SC 1.3.1 - SC 2.4.1 - SC 2.4.6 - SC 2.4.10)</small>
+                        <h2>Looks <span class="gdi">Complicated</span></h2>
+                        <h2>It Isn't!!</h2>
```

Comment:
Can you add a little more advice for how to step through the criteria? Is the best way to browse? search?